### DISCUSSION TITLE
What is the plan for v2?

### DISCUSSION THREAD

**Discussion by maxwelllevin** [ARM-DOE/ACT#742] [General]
I've been noticing an increasing number of issues/PRs with the `v2.0.0` tag. Does this mean that version 2.0 is coming soon? Will there be an announcement before this drops and/or an upgrade guide for when the changes are released?

**Comment by AdamTheisen** [comment]
@maxwelllevin Yes!  We are working on development towards a version 2.0 where some things will break. We were thinking before the end of the year but we have a lot of it staged and may be able to do it in November timeframe.  

We will plan on developing a guide of all the changes.  The main ones are just name changes of files/functions for consistency purposes.  I didn't think about how we will announce this but I will touch base with our communications team.  @mgrover1 @zssherman any thoughts?

**Reply by zssherman** [reply]
@AdamTheisen Are we able to work with ARM communications? Another place we can post as well is in the discussions on github. We use that mostly for the forum. But not sure what other ways of communicating the release

**Reply by mgrover1** [reply]
I think a 2.0 guide sounds reasonable!

**Comment by AdamTheisen** [comment]
@maxwelllevin Just to keep you posted on V2.0, we are looking at an alpha release in the next 2-3 weeks to shake out bugs and gather feedback with a full release planned in January before AMS.  We are also going to merge in the V2.0 guide soon as well so it's available with the planned changes ahead of time.

**Reply by AdamTheisen** [reply]
V2.0 Guide: https://arm-doe.github.io/ACT/userguide/GUIDE_V2.html

**Reply by maxwelllevin** [reply]
Thanks for the heads-up @AdamTheisen!

**Comment by AdamTheisen** [comment]
@maxwelllevin a pre-release of ACT V2.0 is available now to test if you would like.  Instructions are here: https://github.com/ARM-DOE/ACT#version-20

**Comment by maxwelllevin** [comment]
Thanks @AdamTheisen! I'll try it out on some VAPs tomorrow and let you know how it goes.

**Comment by zssherman** [comment]
Going to close this for now.
